### ISSUE TITLE
Renderable  setMaterial always throw SubmeshOutOfRangeException

### ISSUE THREAD

**Issue by KamikX** [google-ar/sceneform-android-sdk#14]
Hi, in Renderable class, method setMaterial(int submeshIndex, Material material) start on line 94, missing else statement.
```
    public void setMaterial(int submeshIndex, Material material) {
        if (submeshIndex < this.materialBindings.size()) {
            this.materialBindings.set(submeshIndex, material);
            this.changeId.update();
        }

        throw this.makeSubmeshOutOfRangeException(submeshIndex);
    }
```

**Comment by malik-at-work** [comment]
Thank you @KamikX ! A fix will be in our next update.

**Comment by malik-at-work** [comment]
Fixed in the 1.3 release.
